### PR TITLE
Cypress test. Collapse sidebar. Fullscreen.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Cypress test for PR #3289 
An additional plugin for Cypress is installed for correct interaction with the fullscreen activation button. 
Otherwise, I would get an error
```
Failed to execute 'requestFullscreen' on 'Element': API can only be initiated by a user gesture.
onClick @ right-group.tsx:150

Uncaught (in promise) TypeError: fullscreen error
    at onClick (cvat-ui.48af1c4a35bbdd68752d.min.js:435)
    at Q (cvat-ui.48af1c4a35bbdd68752d.min.js:24)
    at Object.s (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at p (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at cvat-ui.48af1c4a35bbdd68752d.min.js:352
    at v (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at it (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at at (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at st (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
    at pt (cvat-ui.48af1c4a35bbdd68752d.min.js:352)
```

The test must also be run in the headed mode for the fullscreen to work correctly. To do this, I put it in the directory where the tests are run in the headed mode.
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@dvkruchinin 

Actually, we do not need exactly fullscreen in this test. Just need to resize the browser window anyhow. Is cypress able to do it?
Maybe manual raising a "resize" event would be a solution also.

**Comment by dvkruchinin** [comment]
> Actually, we do not need exactly fullscreen in this test. Just need to resize the browser window anyhow. Is cypress able to do it?

Thanks. I reworked the test using resizing by means of ```viewport()```.

**Comment by bsekachev** [comment]
@dvkruchinin 
``actions_tasks3/case_44_changing_default_value_for_attribute.js`` fails often because of an element detached from DOM. Could you please look into it?

File: tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js [hunk 1]
```diff
@@ -42,13 +53,22 @@ context('Collapse sidebar/appearance', () => {
                     expect(currentValueLeftBackground).to.be.greaterThan(defaultValueLeftBackground);
                 });
 
-            // wait when background fitted
-            cy.wait(500);
+            // Check issue 3250
+            cy.get('#cvat_canvas_content').invoke('attr', 'style').then((canvasContainerStyle) => {
+                cy.viewport(2999, 2999); // Resize window
+                cy.get('#cvat_canvas_content').should('have.attr', 'style').and('not.equal', canvasContainerStyle);
+                cy.viewport(1300, 960); // Return to the original size
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js:60]: Can we get the original size before resizing and apply it after the test? In cypress.json, we can update viewport size, but probably we will not remember about this place. It is like a second place of the truth.

**Comment by dvkruchinin** [comment]
> `actions_tasks3/case_44_changing_default_value_for_attribute.js` fails often because of an element detached from DOM. Could you please look into it?

This is a known problem. Attempts have already been made to solve it. I will create a separate task for further study of the problem.

File: tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js]: Reworked to use values from cypress.json

**Review by bsekachev — APPROVED** [review]

**Pull request merged**